docs/commands.md

```diff
@@ -148,6 +148,26 @@ Here we've required that the president node's name will have the value "Jed Bart
 
 There's no difference between inline predicates and predicates specified within the WHERE clause.
 
+It is possible to specify a pattern predicate to filter upon:
+
+The following queries are equal in their results, and they return all the presidents and the countries they won at:
+
+```sh
+MATCH (p:president), (s:state) WHERE (p)-[:won]->(:s) RETURN p, s
+```
+
+and
+
+```sh
+MATCH (p:president)-[:won]->(s:state) RETURN p, s
+```
+
+Pattern predicate can be also negated or combined with logical operators. The following query asks about all the presidents that did not won in the countries where they were their governors.
+
+```sh
+MATCH (p:president), (s:state) WHERE NOT (p)-[:won]->(:s) AND (p)->[:governor]->(s) RETURN p, s
+```
```

Comment:
This looks good! Just some rewording suggestions (note the changes in queries as well):

> It is also possible to filter on graph patterns. The following queries, which return all presidents and the states they won in, produce the same results:
> 
> ```sh
> MATCH (p:president), (s:state) WHERE (p)-[:won]->(s) RETURN p, s
> ```
> and
> ```sh
> MATCH (p:president)-[:won]->(s:state) RETURN p, s
> ```
> Pattern predicates can be also negated and combined with the logical operators AND, OR, and NOT. The following query returns all the presidents that did not win in the states where they were governors:
> ```sh
> MATCH (p:president), (s:state) WHERE NOT (p)-[:won]->(s) AND (p)->[:governor]->(s) RETURN p, s
> ```